contrib/fsl_integration/sct_plugin.py

```diff
@@ -44,11 +44,11 @@
 import logging
 import webbrowser
 
-import wx
-import wx.lib.agw.aui as aui
-import wx.html as html
+import wx                     # noqa: F821 (wx is provided by the FSLeyes context)
+import wx.lib.agw.aui as aui  # noqa: F821 (wx is provided by the FSLeyes context)
+import wx.html as html        # noqa: F821 (wx is provided by the FSLeyes context)
```

Comment:
Good catch. This isn't actually a flake8 linting issue; it's a PyCharm issue:

> ![image](https://user-images.githubusercontent.com/16181459/228097117-ad5b82f6-fd55-437c-9791-d8c26b90533a.png)

> ![image](https://user-images.githubusercontent.com/16181459/228097075-cbe29ee1-001b-4bd5-8070-7695d692d257.png)

It's complaining that `wx` and `fsleyes` aren't installed to `venv_sct` (in the environment chosen to be PyCharm's interpreter), but as you suggest, this isn't a linting issue at all.

(I mistakenly applied `# noqa` comments to silence the wrong underlying issue.)

Reply:
I think it would still be nice to add notes that silence interpreter warnings, and explain that these packages don't need to be installed inside of `venv_sct`? 

(Perhaps selfishly, as these IDE errors are a pet peeve, but also as a safeguard against future devs seeing the errors and trying to install `fsleyes` into the SCT environment, which has caused major issues in the past.)

Reply:
I see! Yes, accounting for PyCharm warnings is totally fine, for all the reasons you give. I'll re-add the `# noqa: F821` comments for the imports with an explanatory note, then merge this PR.